### PR TITLE
capture2text: Add version 4.6.2

### LINKED ISSUE #3905: [Request] Add Capture2Text
Capture2Text is open-source software that enables users to quickly OCR a portion of the screen using a keyboard shortcut. The resulting text will be saved to the clipboard by default. 

[Homepage](http://capture2text.sourceforge.net/)
[download](https://sourceforge.net/projects/capture2text/files/Capture2Text/)

### PR DESCRIPTION
**Pull request by vuwjzl17985**
- Closes #3905

### CODE REVIEW

File: bucket/capture2text.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    "shortcuts": [
+        [
+            "Capture2Text.exe  --portable",
+            "Capture2Text"
+        ],
+        [
+            "Capture2Text_CLI.exe",
+            "Capture2Text_CLI"
+        ]
+    ],
```
Comment (Ash258) [inline bucket/capture2text.json:32]:
```suggestion
            "Capture2Text.exe",
            "Capture2Text",
            "--portable"
        ]
    ],
    "persist": "Capture2Text",
```

File: bucket/capture2text.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            "hash": "c6d9e9341124c9218df3e9e73f14744baf0128412757bb5ee6decb3ac3625639"
+        },
+        "32bit": {
+            "url": "https://downloads.sourceforge.net/project/capture2text/Capture2Text/Capture2Text_v4.6.2/Capture2Text_v4.6.2_32bit.zip",
+            "hash": "2f7a1bdc13930df681297229ecb0526e2681b74c93ea89b3d77f8fe2f2c4bd68"
+        }
+    },
+    "extract_dir": "Capture2Text",
+    "bin": [
+        [
```
Comment (Ash258) [inline bucket/capture2text.json:18]:
```suggestion
        "Capture2Text_CLI.exe",
        [
```

File: bucket/capture2text.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    "version": "4.6.2",
+    "license": "GPL",
+    "architecture": {
+        "64bit": {
+            "url": "https://cfhcable.dl.sourceforge.net/project/capture2text/Capture2Text/Capture2Text_v4.6.2/Capture2Text_v4.6.2_64bit.zip",
+            "hash": "c6d9e9341124c9218df3e9e73f14744baf0128412757bb5ee6decb3ac3625639"
+        },
+        "32bit": {
+            "url": "https://downloads.sourceforge.net/project/capture2text/Capture2Text/Capture2Text_v4.6.2/Capture2Text_v4.6.2_32bit.zip",
+            "hash": "2f7a1bdc13930df681297229ecb0526e2681b74c93ea89b3d77f8fe2f2c4bd68"
```
Comment (Ash258) [inline bucket/capture2text.json:13]:
```suggestion
            "hash": "sha1:3320399d755fbbbdc10467ff2d5e2e6110ea9e54"
```

File: bucket/capture2text.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
+{
+    "homepage": "http://capture2text.sourceforge.net/",
+    "description": "Capture2Text enables users to quickly OCR a portion of the screen using a keyboard shortcut. The resulting text will be saved to the clipboard by default.",
+    "version": "4.6.2",
+    "license": "GPL",
+    "architecture": {
+        "64bit": {
+            "url": "https://cfhcable.dl.sourceforge.net/project/capture2text/Capture2Text/Capture2Text_v4.6.2/Capture2Text_v4.6.2_64bit.zip",
+            "hash": "c6d9e9341124c9218df3e9e73f14744baf0128412757bb5ee6decb3ac3625639"
```
Comment (Ash258) [inline bucket/capture2text.json:9]:
```suggestion
            "hash": "sha1:c92444ebbbaa6c9e8c05b8961242434d5470e379"
```

File: bucket/capture2text.json [hunk 5]
```diff
@@ -0,0 +1,47 @@
+{
+    "homepage": "http://capture2text.sourceforge.net/",
+    "description": "Capture2Text enables users to quickly OCR a portion of the screen using a keyboard shortcut. The resulting text will be saved to the clipboard by default.",
+    "version": "4.6.2",
+    "license": "GPL",
```
Comment (Ash258) [inline bucket/capture2text.json:5]:
```suggestion
    "version": "4.6.2",
    "description": "OCR capture utility",
    "homepage": "http://capture2text.sourceforge.net/",
    "license": "GPL-3.0-only",
```

**Comment by vuwjzl17985** [comment]
Update changes as requested. Thank you very much @Ash258 .

**Pull request merged**